.github/workflows/integration.yml

```diff
@@ -274,6 +274,30 @@ jobs:
       - name: run proposals tests
         run: yarn test
         working-directory: a3p-integration
+      - name: collect all core proposals
```

Comment:
Proposals are what are sent to stakers

These are core eval scripts, which can be included in a proposal, but not proposals themselves